lntest/node.go

```diff
@@ -1331,7 +1249,7 @@ func (hn *HarnessNode) cleanup() error {
 // Stop attempts to stop the active lnd process.
 func (hn *HarnessNode) stop() error {
 	// Do nothing if the process is not running.
-	if hn.processExit == nil {
+	if hn.runCtx == nil {
```

Comment:
I don't think that we set the ctx ever to null, maybe there is another way to check if we already called `stop()`?

Reply:
`runCtx` will be nil if `start` is not called. So it's checking whether there exists a context or not. If not, there's no need and we cannot run the code inside `stop`.